stages/14-director-gui-check/gui-dmaas-device-aws

```diff
@@ -93,5 +92,7 @@ cat ~/.dmaas/aws_cred ~/.dmaas/minio_cred >> cred.ini
 
 bash scripts/dop-gui-stages/run-gui-tests.sh ${GROUP} ${GUID} ${URL} ${THREADS} eu-north-1
 
-# cp  $path/.awscluster2/config ~/.kube/config
-# bash $path/utils/e2e-cr jobname:tcid-dir-dmaas-schd-local-dev-aws-bucket jobphase:Completed
\ No newline at end of file
+cp  $path/.awscluster2/config ~/.kube/config
```

Comment:
Please explain what is being done here

Reply:
here we are connecting to cluster 2

Reply:
Yes. Please put all the responses as code comments & then merge this.